packages/big-design/src/components/Modal/Modal.tsx

```diff
@@ -16,11 +16,15 @@ import {
 } from './styled';
 
 export interface ModalProps {
+  actions?: React.ReactNode;
   backdrop: boolean;
-  isOpen: boolean;
   closeOnClickOutside: boolean;
   closeOnEscKey: boolean;
+  header?: string;
+  isOpen: boolean;
   variant: 'modal' | 'dialog';
+  withActionsBorder?: boolean;
```

Comment:
Do we have multiple designs? I think this should be consistent in all modals and not configurable.

Reply:
@deini It seems like you added it in #133 to support @animesh1987 use case. Is this something we need to touch back with design?

Reply:
Removed borders.